internal/cli/views/basics/form/form.go

```diff
@@ -0,0 +1,212 @@
+package form
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	btn "iwakho/gopherkeep/internal/cli/views/basics/button"
+	"iwakho/gopherkeep/internal/model"
+
+	"github.com/charmbracelet/bubbles/cursor"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+var (
+	focusedStyle        = btn.FocusedStyle
+	cursorStyle         = btn.FocusedStyle
+	noStyle             = lipgloss.NewStyle()
+	helpStyle           = btn.BlurredStyle
+	cursorModeHelpStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("244"))
+)
+
+type Control interface {
+	Submit(p model.Pair) error
+}
+
+type ModelForm struct {
+	Name        string
+	focusIndex  int
+	inputs      []textinput.Model
+	cursorMode  cursor.Mode
+	NextPage    func()
+	buttons     []string
+	indexMax    int
+	failMessage string
+	isUpdated   bool
+	Control
+}
+
+type FormCaller struct {
+	FormName    string
+	InputNames  []string
+	ButtonNames []string
+}
+
+func InitForm(fc FormCaller) ModelForm {
+	m := ModelForm{
+		Name:   fc.FormName,
+		inputs: make([]textinput.Model, len(fc.InputNames)),
+	}
+	m.buttons = fc.ButtonNames
+	var t textinput.Model
+	for i := range m.inputs {
+		t = textinput.New()
+		t.Cursor.Style = cursorStyle
+		t.CharLimit = 32
+
+		if i == 0 {
+			t.Placeholder = fc.InputNames[i]
+			t.Focus()
+			t.PromptStyle = focusedStyle
+			t.TextStyle = focusedStyle
+		} else {
+			t.Placeholder = fc.InputNames[i]
+			t.EchoMode = textinput.EchoPassword
+			t.EchoCharacter = '•'
+		}
+		m.inputs[i] = t
+	}
+	m.indexMax = len(m.inputs) + len(m.buttons) - 1
+	return m
+}
+
+func (m ModelForm) Init() tea.Cmd {
```

Comment:
В качестве ресивера тоже можно использовать указатель.
```suggestion
func (m *ModelForm) Init() tea.Cmd {
```